trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -155,7 +162,7 @@ const getKeyingSheetAdjustmentViewModel = (adjustment: KeyingSheetAdjustment | n
     return { amount: undefined, change: 'NONE' };
   }
   return {
-    amount: adjustment.amount.toFixed(2),
+    amount: getFormattedMonetaryValue(adjustment.amount),
```

Comment:
Do we have E2E tests for this? The formatting has changed so i'd expect some E2E tests to fail, otherwise LGTM

Reply:
Thanks, just spotted that these were failing

Just pushed a commit to address this thanks - the e2e tests now test for the comma (thousands separator) and the two decimal places